docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -1,86 +1,90 @@
 # Extending a plug-in
+
 Before you begin, be sure to complete the [Installing the sample plug-in](cli-installing-sample-plugin.md) tutorial.
 
 ## Overview
 
-This tutorial demonstrates how to extend the plug-in that is bundled with this sample by:
-
-1. Creating a Typescript interface for the Typicode response data
-2. Creating a programmatic API
-3. Creating a command definition
-4. Creating a command handler
+This tutorial demonstrates how to extend the plug-in that is bundled with this sample.
 
-We'll do this by using `@zowe/imperative` infrastructure to surface REST API data on our Zowe&trade; CLI plug-in.
+Specifically, the tutorial shows data from [this URI](https://jsonplaceholder.typicode.com/todos) by [Typicode](https://jsonplaceholder.typicode.com/). Typicode serves sample REST JSON data for testing purposes. To do this, use the `@zowe/imperative` infrastructure to surface REST API data on our Zowe&trade; CLI plug-in. **[What is "this" in "To do this ..."? Can we use another verb other than "surface"?]**
```

Comment:
@adam-wolfe please respond

Reply:
Could say something like: "This tutorial will show you how build a Zowe CLI Plug-in that interacts with the Typicode sample REST API, which can help guide you through the steps of building a CLI Plug-in for any REST API."

or maybe:

"This tutorial will use the Typicode REST API as a guide for how to build a Zowe CLI plug-in that interacts with REST APIs on the mainframe."

I'm not sure how it should be worded, but the takeaway is that this is a tutorial for building a Zowe CLI plug-in that interacts with a generic REST API, and that a developer could adapt the information here to building their own plug-in for a product.